packages/v4/package.json

```diff
@@ -22,7 +22,7 @@
     "@patternfly/react-catalog-view-extension": "4.95.2",
     "@patternfly/react-docs": "5.103.66",
     "@patternfly/react-log-viewer": "4.87.100",
-    "@patternfly/react-topology": "^4.91.26",
```

Comment:
This does not look correct. You don't want to remove topology in place of console

Reply:
Ah yep, not sure if I mixed something up resolving merge conflicts during a rebase or what. Great catch.